fonts.css

```diff
@@ -0,0 +1,26 @@
+@font-face {
+  font-family: 'MuseoSansCyrl';
+  font-display: swap;
+  src: url("../fonts/MuseoSansCyrl.eot");
+  src: 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/baliproject/pullRequest?prid=5558708)